docs/_includes/lhn-template.html

```diff
@@ -0,0 +1,44 @@
+<ul class="lhn-items">
+  <li class="home-link">
+    <div class="selected">Home</div>
```

Comment:
Oh I noticed the `Home` is selected in all pages:

<img width="418" alt="Screen Shot 2022-10-31 at 13 45 14" src="https://user-images.githubusercontent.com/6829422/199097073-fe096233-6b57-44c2-af50-d27caffbcfe9.png">

I think we can do this make it selected only in the homepage and clickable for other pages:

```suggestion
    {% if page.lhn_content == 'homepage' %}
      <div class="selected">Home</div>
    {% else %}
      <a href="/main">Home</a>
    {% endif %}
```

Reply:
good catch